packages/manager/modules/pci/src/projects/project/storages/databases/database/users/users.routing.js

```diff
@@ -186,6 +192,10 @@ export default /* @ngInject */ ($stateProvider) => {
 
         guideUrl: () => null,
         hideRolesMatrix: () => true,
+        onDestroy: /* @ngInject */ (DatabaseService, users, database) => () =>
+          users.forEach((u) =>
+            DatabaseService.stopPollingUserStatus(database.id, u.id),
```

Comment:
We avoid to use the shorter variable name.
 ```suggestion
          users.forEach((user) =>
            DatabaseService.stopPollingUserStatus(database.id, user.id),
```

Reply:
Changed: https://github.com/ovh/manager/pull/5804/commits/e834361999d86a502ef3c7f30777ba0d707cb5d9